backend/app/views/spree/admin/shared/linkables/_spree_product.html.erb

```diff
@@ -0,0 +1,23 @@
+<% if resource.linked_resource %>
+   <% resorce_holder = [resource.linked_resource] %>
+   <% id_holder = :id %>
+   <% name_holder = :name %>
+   <% active_id_holder = resource.linked_resource.id %>
+ <% end %>
+ <%= f.field_container :linked_resource_id, class: ['form-group'] do %>
+   <%= f.label :linked_resource_id, Spree.t('product') %>
+   <%= f.select :linked_resource_id, options_from_collection_for_select(resorce_holder || [] , id_holder, name_holder, active_id_holder || nil), { include_blank: true }, class: 'autocomplete' %>
+   <%= f.error_message_on :linked_resource_id %>
+   <small class="form-text text-muted"><%= resource.link %></small>
+<% end %>
+
+<script>
+  document.addEventListener('DOMContentLoaded', function() {
```

Comment:
@MatthewKennedy can we have one autocomplete setup per app and get the values for `data_attrbute_name`, `apiUrl` etc from `data-` HTML tags? this will help us later move gradually to Stimulus and Hotwire :)

Reply:
Yeah that works for me, I'll sort that, I prefer that approach myself.

Do you want that change on a fresh PR?

Reply:
Actually no, I'll do it on this PR then I can implement it straight onto the cms_pages if that's okay?